packages/components/src/ui/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './grid';
```

Comment:
We should include `./font-size-control` here but at the moment types don't build because those files aren't typed.

Reply:
can we just include subfolders of `ui` rather than including the whole thing in the `includes`?

Reply:
Right, we can 👍